src/cli-constant.js

```diff
@@ -66,6 +67,13 @@ Available options:
   --write                  Edit the file in-place. (Beware!)
   --list-different or -l   Print filenames of files that are different from Prettier formatting.
   --config                 Path to a prettier configuration file (.prettierrc, package.json, prettier.config.js).
+  --config-precedence <cli-override|file-override|prefer-file>
+                           Defines how config file should be evaluated in combination of CLI arguments
+                           cli-override  | default config => config file => CLI overrides
+                           file-override | default config => CLI overrides => config file
```

Comment:
Maybe we should say `CLI options` instead of `CLI overrides` to avoid confusion.

Reply:
That sounds reasonable to me